cylc/uiserver/resolvers.py

```diff
@@ -321,6 +324,76 @@ async def play(cls, workflows, args, workflows_mgr, log):
         await workflows_mgr.scan()
         return response
 
+    @classmethod
+    async def enqueue(cls, stream, queue):
```

Comment:
We can get away with a static method here:

```suggestion
    @staticmethod
    async def enqueue(stream, queue):
```